lib/workers/repository/onboarding/branch/check.js

```diff
@@ -32,12 +32,13 @@ const isOnboarded = async config => {
   return false;
 };
 
-const onboardingPrExists = config =>
-  platform.findPr(
+const onboardingPrExists = async config => {
+  const pullRequest = await platform.findPr(
     `${config.branchPrefix}configure`,
-    'Configure Renovate',
-    'open'
+    'Configure Renovate'
   );
+  return pullRequest && !pullRequest.isClosed ? pullRequest : null;
+};
```

Comment:
I think these method changes can all be reverted, now that we have normalized the 'open'/'closed' state.